ext/pgsql/pgsql.c

```diff
@@ -147,6 +147,17 @@ ZEND_TSRMLS_CACHE_DEFINE()
 ZEND_GET_MODULE(pgsql)
 #endif
 
+struct _pcre_cache_entry {
```

Comment:
Please don't import internals from another extension into this extension. I added some suggestions on how to avoid this.